dockerdb/constants.py

```diff
@@ -0,0 +1,18 @@
+from os import getenv
+
+from dotenv import load_dotenv
+from faker import Faker
+
+load_dotenv()
+
+fake = Faker()
+
+PG_DATABASE = getenv(key="PG_DATABASE", default="piccolo_database")
+PG_HOST = "localhost"
+PG_IMAGE_NAME = getenv(key="POSTGRES_IMAGE_NAME", default="postgres:latest")
+PG_PASSWORD = getenv(key="PG_PASSWORD", default=fake.password())
```

Comment:
I'm not sure about using faker here - just because it'll create the database with this random password, but the user won't know what it is. Maybe hardcode the default to something else instead?

Reply:
Because it's an env variable you can see it in the docker desktop

Reply:
Cool - didn't think of that!

Reply:
Do you have a specific checklist of boilerplate code required for pypi release?

Reply:
Probably best just to copy over the `setup.py` and `.github/workflows/release.yaml` file from the main Piccolo repo, and tweak as appropriate.

Reply:
Cool, I'll do that. Thanks!